syntax = "proto2";

package settings;

message Point3D {
    required float x = 1;
    required float y = 2;
    required float z = 3;
}
